src/styles/_catalog.scss

```
@@ -1,22 +1,37 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(4, 199px);
+  gap: 48px;
+  justify-content: center;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: 199px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (max-width: 1024px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: repeat(3, 199px);
+  }
+}
```

Use nesting. Move media query into `.catalog` selector. `display: grid;` is redudant here
```suggestion
@media (max-width: 1024px) {
  grid-template-columns: repeat(3, 199px);
}
```